api/src/list/list.controller.ts

```diff
@@ -184,17 +157,13 @@ export class ListsController {
 
   @UseInterceptors(RemoveListFieldsInterceptor)
   @UseGuards(ListAuthGuard, ListPrivacyAuthGuard)
-  @Patch('/:id/updatePrivacy')
+  @Patch('/:id/update')
```

Comment:
Is this supposed to update any prop in a list? If so, we should go with

```js
@Patch('/:id')
```

If not, we should mention what this is updating.